clang/test/Driver/sycl-target-mismatch.cpp

```diff
@@ -22,4 +22,7 @@
 // RUN: %clangxx -fsycl -fsycl-targets=spir64_gen %S/Inputs/SYCL/liblin64.a \
 // RUN:   -Wno-sycl-target -### %s 2>&1 \
 // RUN:  | FileCheck %s -check-prefix=SPIR64_DIAG
+// RUN: %clangxx -fsycl -fsycl-targets=spir64_gen %S/Inputs/SYCL/liblin64.a \
```

Comment:
I think passing `%S/Inputs/SYCL/liblin64.a` and `###` is not necessary here.

Reply:
The documentation doesn't show a full command line, it just demonstrates how the options work together - it probably should be updated to be a little clearer.

The use of -### ensures that only the driver is invoked here.  Without it, we would run additional tools that aren't necessary since we are only checking a diagnostic from the driver.